.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,68 @@
+name: PR Build
+
+on:
+    pull_request:
+        branches: [main]
+
+jobs:
+    build-webui:
+        name: Building Webui
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+
+            - name: Generate typescript openapi client using gradle
+              run: |
+               gradle generateTypeScriptClient -PsourceMaven=https://development.galasa.dev/main/maven-repo/framework

```

Comment:
Please update this to `https://development.galasa.dev/gh/maven-repo/framework`

Reply:
done